templates/hiera.yaml.erb

```diff
@@ -5,12 +5,12 @@
   @backends.unshift('eyaml')
   @backends = @backends.uniq
 end -%>
-<%= @backends.to_yaml.split("\n")[1..-1].map{|e| "  #{e.strip}"}.join("\n") %>
+<%= @backends.map{|x| "  - #{x}"}.join("\n") %>
 
 :logger: <%= @logger %>
 
 :hierarchy:
-<%= @hierarchy.to_yaml.split("\n")[1..-1].map{|e| "  #{e.strip}"}.join("\n") %>
```

Comment:
Using `to_yaml` here looked a bit convoluted anyway...

`[1..-1]` to throw away the `---`
`map{|e| "  #{e.strip}"}` to reindent